src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,51 @@
+.nav {
+  &__list {
+    display: flex;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
+  
+  &__item {
+    margin-left: 20px;
+
+    &:first-child {
+      margin-left: 0;
+    }
+  }
+
+  &__link {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 60px;
+    text-decoration: none;
+    text-transform: uppercase;
+    color: #000;
+    display: block;
+    position: relative;
+
+    &::after {
+      content: "";
+      position: absolute;
+      background-color: #00acdc;
```

Comment:
You may try to create a variable for this color. It's also seen often in the code.